docs/guides/staging-patches.md

```diff
@@ -0,0 +1,122 @@
+# If auto_update: false, you will need to use package:shorebird_code_push to trigger updates.
+# https://pub.dev/packages/shorebird_code_push
+# Uncomment the following line to disable automatic updates.
+# auto_update: false
+```
+
+## Create a release
+
+Now that we've created our apps on shorebird, we need to create releases (one for each platform). To create a release, we'll use the `shorebird release` command.
+
+```sh
+# Create a release for Android
+shorebird release android
+
+# Create a release for iOS
+shorebird release ios-alpha
+```
+
+We can verify the releases were created successfully by visiting [Shorebird console](https://console.shorebird.dev).
+
+You should also [submit the generated app bundles to the Play Store](/guides/release/android#upload-to-the-play-store) and [submit the generated ipa to the App Store](/guides/release/ios#upload-to-the-app-store).
+
+## Creating a patch
+
+Now that we have our releases on the Play Store and App Store, we can create a patch using `shorebird patch`. For the sake of this example, let's set the `backgroundColor` of the `Scaffold` to `Colors.cyan` in `lib/main.dart`:
+
+```diff
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(const MainApp());
+}
+
+class MainApp extends StatelessWidget {
+  const MainApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const MaterialApp(
+      home: Scaffold(
++       backgroundColor: Colors.cyan,
+        body: Center(
+          child: Text('Hello World!'),
+        ),
+      ),
+    );
+  }
+}
+
+```
+
+Now that we've applied the changes, let's create a patch:
+
+```sh
+# Stage a patch on Android
+shorebird patch android --staging
+
+# Stage a patch on iOS
+shorebird patch ios-alpha --staging
+```
+
+## Preview the release
+
+Next, preview the app release locally on a device or emulator, use `shorebird preview`.
+
+```sh
+# Preview the release in the staging environment
+shorebird preview --staging --app-id ee322dc4-3dc2-4324-90a9-04c40a62ae76 --release-version 1.0.0+1
+```
+
+Shorebird will download the release and run it on your device in the staging environment.
+
+The first time the app is re-launched, we should still see the white `Scaffold` and shorebird will detect and install the patch in the background. Kill and re-launch the app a second time to see the applied patch with the cyan `Scaffold` background.
+
+If all went well, you should see the patch was applied after re-launching the app a second time. Congrats, you've validated your patch in the staging environment 🥳
+
+## Promote the patch
+
+Now that you have validated the patch, you can push the patch to all devices by promoting it to production from the [Shorebird console](https://console.shorebird.dev). Navigate to the release details page and click on the "Go Live" button.
+
+![go-live](https://github.com/shorebirdtech/docs/assets/8855632/70fa5176-4e6a-4d7a-b820-6b24eca2c27a)
```

Comment:
Will this image persist?  I'm not sure what "/assets" does on github?

Reply:
I believe so it's just GitHub's static assets API (same one it uses when you upload images to issues/PRs)